### PR TITLE
cloud: updated dscription about getting gcp gsutil uri

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Updated the way to get the GCP gsutil URI and added screenshots.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 1]
```diff
@@ -147,12 +147,16 @@ To allow TiDB Cloud to access the source data in your GCS bucket, you need to co
     >
     > To remove the access to TiDB Cloud, you can simply remove the access that you have granted.
 
-6. On the **Bucket details** page, click the **CONFIGURATION** tab, and then copy your GCS bucket URI from the **gsutil URI** field.
+6. On the **Bucket details** page, click the **OBJECTS** tab. If you want to copy a file's gsutil URI, select the file, click **Open object overflow menu**, and then click **Copy gsutil URI**, as shown in the screenshot below.
```
Comment (TomShawn) [inline tidb-cloud/config-s3-and-gcs-access.md:150]:
```suggestion
6. On the **Bucket details** page, click the **OBJECTS** tab.

    If you want to copy a file's gsutil URI, select the file, click **Open object overflow menu**, and then click **Copy gsutil URI**, as shown in the screenshot below.
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 2]
```diff
@@ -147,12 +147,16 @@ To allow TiDB Cloud to access the source data in your GCS bucket, you need to co
...
 
-    ![Get bucket URI](/media/tidb-cloud/gcp-bucket-url.png)
+    ![Get bucket URI](/media/tidb-cloud/gcp-bucket-uri01.png)
 
-7. In the TiDB Cloud console, go to the **Data Import** page where you get the Google Cloud Service Account ID, and then paste the GCS bucket URI to the **Bucket URI** field. Note that you must add `/` to the end of the URI.
+    If you want to use a folder's gsutil URI, open the folder, and then click the copy button following the folder name to copy the folder name, as shown in the screenshot below. After that, you must add `gs://` to the beginning and `/` to the end of the name to get a correct URI of the folder.
```
Comment (TomShawn) [inline tidb-cloud/config-s3-and-gcs-access.md:154]:
```suggestion
    If you want to use a folder's gsutil URI, open the folder, and then click the copy button following the folder name to copy the folder name. After that, you need to add `gs://` to the beginning and `/` to the end of the name to get a correct URI of the folder.
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 3]
```diff
@@ -147,12 +147,18 @@ To allow TiDB Cloud to access the source data in your GCS bucket, you need to co
...
+    ![Get bucket URI](/media/tidb-cloud/gcp-bucket-uri01.png)
 
-    For example, if your bucket URI is `gs://tidb-cloud-source-data`, you need to fill in `gs://tidb-cloud-source-data/`.
+    If you want to use a folder's gsutil URI, open the folder, and then click the copy button following the folder name to copy the folder name. After that, you need to add `gs://` to the beginning and `/` to the end of the name to get a correct URI of the folder.
+
+    For example, if the folder name is `tidb-cloud-source-data`, you need to use `gs://tidb-cloud-source-data/` as the URI.
+
+    ![Get bucket URI](/media/tidb-cloud/gcp-bucket-uri02.png)
+
+7. In the TiDB Cloud console, go to the **Data Import** page where you get the Google Cloud Service Account ID, and then paste the GCS bucket URI to the **Bucket URI** field. The following is an example.
```
Comment (Frank945946) [inline tidb-cloud/config-s3-and-gcs-access.md:162]: and then paste the GCS bucket URI to the **Bucket URI** field. 
change to
and then paste the GCS bucket gsutil URI to the **Bucket gsutil URI** field.

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/config-s3-and-gcs-access.md (hunk 3 above)
Comment (hfxsd) [inline tidb-cloud/config-s3-and-gcs-access.md:162]:
```suggestion
7. In the TiDB Cloud console, go to the **Data Import** page where you get the Google Cloud Service Account ID, and then paste the GCS bucket gsutil URI to the **Bucket gsutil URI** field. The following is an example.
```

**Pull request merged**